avatars/avatars.js

```diff
@@ -393,8 +393,9 @@ const _makeDebugMesh = (avatar) => {
 
 
 
-class Avatar {
+class Avatar extends EventTarget {
```

Comment:
Seems like `Avatar` is not emitting events so it does not need to be an `EventTarget`?

Reply:
I see the helper emits events: https://github.com/webaverse/app/pull/2804/files#diff-94f53fcba17d684d2a2411861f461b371bd86726f85126685b76dbf4fa63717eR1186

IMO this isn't good though, if `Avatar` needs to be emitting events it should be an `Avatar` method doing it.

Reply:
Movec dispatchEvent() from animationHelper.js to avatars.js. https://github.com/webaverse/app/pull/2804/commits/fba731c118e8aa17ef0e813a8b7dcfe6405bd17d